index.html

```diff
@@ -1824,6 +1830,67 @@ <h3 id="applying">
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          <code title="">background_color</code> member
+        </h3>
+        <p>
+          The <dfn id=
+          "member-background_color"><code>background_color</code></dfn> member
+          describes the expected background color of the web application. It
+          will repeat what is already available in the application stylesheet
```

Comment:
"will repeat" sounds like the UA should do something. This might confused devs. Just say "repeats".